website/docs/docs/get-started/connect-your-database.md

```diff
@@ -30,6 +30,9 @@ The following fields are required when creating a Postgres, Redshift, or AlloyDB
 | Host Name | The hostname of the Postgres, Redshift, or AlloyDB database to connect to. This can either be a hostname or an IP address. | `xxx.us-east-1.amazonaws.com` |
 | Port | Usually 5432 (Postgres) or 5439 (Redshift) | `5439` |
 | Database | The logical database to connect to and run queries against. | `analytics` |
+
+**Note**: When you set up a Redshift or Postgres connection in dbt Cloud, SSL-related parameters aren't available as inputs.
```

Comment:
✨ 
<img width="826" alt="image" src="https://user-images.githubusercontent.com/17352478/196617384-b8790014-7722-49bf-bc07-6007751f2d5c.png">

Reply:
thank you!